### PR TITLE
Cookie configuration

### PR DESCRIPTION
**Pull request by KaiNissen**
This PR takes care of wmde/fundraising#1298, which is part of [phab:T174683](https://phabricator.wikimedia.org/T174683).

### CODE REVIEW

File: src/Infrastructure/ConfiguredCookie.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+namespace WMDE\Fundraising\Frontend\Infrastructure;
+
+use Symfony\Component\HttpFoundation\Cookie;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Kai Nissen < kai.nissen@wikimedia.de >
+ */
+class ConfiguredCookie {
```
Comment (JeroenDeDauw) [inline src/Infrastructure/ConfiguredCookie.php]: CookieBuilder is probably a better name, since this itself is not a cookie

Comment (KaiNissen) [reply src/Infrastructure/ConfiguredCookie.php]: Naming things... Thank you, just amended the commit.

**Comment by gbirke** [comment]
I'm not sure if we really need to make the cookie settings configurable - what would be a scenario where we'd want to change the settings? And with `cookieBuilder` we already have a central entry point for creating cookies with the right settings. If you don't have objections I'd like to do a followup where I "hard-code" the cookie settings and remove the config.

**Comment by KaiNissen** [comment]
No objections from my side. I can't think of a case in which we'd want to change the settings ever again.

**Comment by JeroenDeDauw** [comment]
Sounds reasonable to me too

**Pull request merged**